src/utilities/analyticsElement.ts

```diff
@@ -6,8 +6,8 @@ interface Sources {
   DOM: MainDOMSource;
 }
 
-interface AnalyticsTrackSources {
-  link: Element;
+export interface AnalyticsTrackSources {
+  link?: Element;
```

Comment:
https://github.com/seibii/clu-clu/pull/13/files#diff-b2f8a4ad2fbd92affd191b70c7db622d8021d361c6478f4b966087b1432c78afR24
`link` is undefined if the element is `a` tag.
The type of 'link' should be Element or undefined.